react/Payment.tsx

```diff
@@ -54,16 +63,31 @@ const Payment: React.FC = () => {
   }
 
   return (
-    <div>
-      <iframe
-        title="card-form-ui"
-        width="400px"
-        height="300px"
-        src={iframeURL}
-        onLoad={() => setupIframe()}
-        ref={iframeRef}
-      />
-      {cardData && <p>{JSON.stringify(cardData)}</p>}
+    <div className="relative">
+      {iframeLoading && (
+        <div className="absolute top-0 left-0 right-0 bottom-0 bg-white-70 z-1 flex items-center justify-center">
+          <Spinner />
+        </div>
+      )}
+      <div className="flex-none">
+        <div className="">
```

Comment:
is this className needed?
```suggestion
        <div>
```